pkg/app/kafka.go

```diff
@@ -370,49 +435,133 @@ func K8SServicePortForward(ctx context.Context, svcName string, ns string, pPort
 	return f, nil
 }
 
-func consume(ctx context.Context, topic string, namespace string) (int, error) {
-	partition := 0
-	forwarder, err := K8SServicePortForward(ctx, bootstrapServerHost, namespace, bootstrapServerPort)
+// createConsumerGroup creates a consumer group in kafka cluster
+func createConsumerGroup(uri string) error {
+	data := Payload{
+		Name:                     topic + "-consumer",
+		AutoOffsetReset:          "earliest",
+		Format:                   "json",
+		EnableAutoCommit:         true,
+		FetchMinBytes:            512,
+		ConsumerRequestTimeoutMs: 30000,
+	}
+	payloadBytes, err := json.Marshal(data)
 	if err != nil {
-		log.Print("error getting forwarder")
-		return 0, err
+		return err
+	}
+	body := bytes.NewReader(payloadBytes)
+	req, err := http.NewRequest("POST", uri+"/consumers/"+topic+"-consumer-group", body)
+	if err != nil {
+		return err
 	}
+	req.Header.Set("Content-Type", "application/vnd.kafka.v2+json")
 
-	defer forwarder.Close()
-	ports, err := forwarder.GetPorts()
+	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return 0, err
+		return err
+	}
+	defer resp.Body.Close()
+	bytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
 	}
-	uri := "localhost:" + fmt.Sprint(ports[0].Local)
+	log.Print(string(bytes))
+	return nil
+}
+
+// Subscribe subscribes a consumer to topic provided in kafka cluster
+func subscribe(uri string) error {
+	data := SubscriptionPayload{
+		Topics: []string{topic},
+	}
+	payloadBytes, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+	body := bytes.NewReader(payloadBytes)
+
+	req, err := http.NewRequest("POST", uri+"/consumers/"+topic+"-consumer-group/instances/"+topic+"-consumer/subscription", body)
```

Comment:
Let's declare the URL format as const and construct string from the format here. This applies to other places as well
```
const subscriptionURLPathFormat = "/consumers/%s-consumer-group/instances/%s-consumer/subscription"
```